docs/conf.py

```diff
@@ -107,28 +129,37 @@
 
 # -- Options for HTML output ----------------------------------------------
 
-
-html_theme = 'scanpydoc'
-html_theme_options = dict(
-    navigation_depth=4,
-    logo_only=True,
-    docsearch_index='scanpy',
-    docsearch_key='fa4304eb95d2134997e3729553a674b2',
-)
-html_context = dict(
-    display_github=True,  # Integrate GitHub
-    github_user='theislab',  # Username
-    github_repo='scanpy',  # Repo name
-    github_version='master',  # Version
-    conf_py_path='/docs/',  # Path in the checkout to the docs root
-)
+html_theme = "sphinx_book_theme"
+html_theme_options = {
+    "repository_url": repository_url,
+    "use_repository_button": True,
+    "logo_only": True,
+    "show_toc_level": 4,  # show all levels in the sidebar
+}
 html_static_path = ['_static']
+html_css_files = ["css/override.css"]
 html_show_sphinx = False
 html_logo = '_static/img/Scanpy_Logo_BrightFG.svg'
+html_title = "scanpy"
 
 
 def setup(app):
-    app.warningiserror = on_rtd
+    """App setup hook."""
+    # TODO: fix all warnings in a future PR
+    # Many come from the tutorials, like the workshop directory
+    # which is not included in the docs
+    #     app.warningiserror = on_rtd
```

Comment:
Wait a second, how could I miss this.

This is an absolute no-go. This makes it very hard to go back to functional docs where nothing is broken, since broken stuff can accumulate after this.

Someone needs to spend far too much time to fix this, and I fear it’s going to be me. Can we please never do this again?